Adding General Motors

johntheholman opened the pull request:
- Adds gm.com as a site that offers Multi-Factor Authentication
- Methods: Email, SMS and Software Auth
- This is a vehicle company so I have it in the Other and Transport categories because there is no "Vehicle" category.

Commit:
Adding General Motors
- Adds gm.com as a site that offers Multi-Factor Authentication
- Methods: Email, SMS and Software Auth
- This is a vehicle company so I have it in the Other and Transport categories because there is no vehicle category.

hkamran80 commented on entries/g/gm.com.json:
```diff
@@ -0,0 +1,22 @@
+    "notes": "Automatically turned on for email. You are only allowed to have one method turned on at a time.",
+    "categories": [
+      "transport",
+      "other"
```
```suggestion
      "transport"
```

I would place it only in `transport`

johntheholman replied:
Thanks @hkamran80. I have updated the category to `transport` only.

Commit:
Updates to PR
Added new SVG logo to img directory
Updated categories to only reference transport

Commit:
Update for test since the SVG is the default value

hkamran80 reviewed (commented):
Can you provide evidence of 2FA? It can be the account settings, the login flow, or the setup flow.
